serverless/unsubscribe-digest/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  init,
+  getUnsubscribeList,
+  sendEmailAndUpdateUnsubscribers,
+} from './unsubscribe'
+
+const handler = async (): Promise<{ statusCode: number }> => {
+  try {
+    await init()
+
+    // Retreive unsubscribed recipients grouped by campaigns and users
+    const unsubscribeDigests = await getUnsubscribeList()
+
+    // generate email digest and send email to each user
+    for (const userDigest of unsubscribeDigests) {
+      await sendEmailAndUpdateUnsubscribers(userDigest)
```

Comment:
why is this in a for loop

Reply:
`unsubscribeDigests` returns `{ email: [ { campaign, unsubscribers }, ...] }`, hence this iterating through to send the email digest and subsequently update the `unsubscribers` table for each user. I considered updating the `usubscribers` separately from this, but thought any error occurred in between could possibly cause the `unsubscribers` table to be not updated for emails already sent out.